docs/testrunner-toolkit/faqs.md

```diff
@@ -36,55 +29,15 @@ To learn more about:
 All images are hosted on Docker Hub.
 
 * [Base image](https://hub.docker.com/r/saucelabs/testrunner-image/tags)
-is called `testrunner`. It contains the tooling necessary to record videos, VNC etc. Plus Chrome, and a Firefox version.
-
+is called `testrunner`. It contains the tooling necessary to record videos, VNC etc., plus Chrome, and a Firefox version.
 * [Base image + Playwright](https://hub.docker.com/r/saucelabs/stt-playwright-jest-node/tags)
-contains saucectl with different versions of Playwright.
-
+contains `saucectl` with different versions of Playwright.
```

Comment:
cypress/playwright/testcafe/puppeteer runners no longer contain `saucectl`. suggest to remove saucectl here.

Reply:
And playwright runner docker page should be this 
```
https://hub.docker.com/r/saucelabs/stt-playwright-node/tags
```

Reply:
Thanks @tianfeng92. I am considering moving these docker hub image links to the docker supported frameworks table on the SC home page and deleting the FAQ page altogether. What do you think?

Reply:
@sweeneyskirt-sl agree. FAQ seems to be redundant after most of documentations added